com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1183,6 +1182,19 @@ public void RefreshSimulationPipelineParameters(NetworkTypeConfiguration configu
             parameter->FuzzOffset = configuration.PacketFuzzOffset;
         }
 
+        internal SimulatorUtility.Parameters GetSimulatorParameters()
```

Comment:
What is the purpose of that method? That it's using `GetWritablePipelineParameters` would tend to indicate that it's meant to get a modifiable version of the parameters. But that's not what it's doing. By dereferencing the pointer and assigning the value to `parameters`, it's actually only doing a copy of current parameters. If that's actually the goal of this method, then `m_NetworkSettings.GetSimulatorStageParameters` will do that without requiring using an internal UTP method.

Reply:
I was not aware of this. This is great. It's also only for the tests at the moment to be sure what we are doing is correct.